docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+[[synthetics-analyze]]
+= Analyze data from synthetic monitors
```

Comment:
**This is a new page that needs review.** It's contained to the [`docs/en/observability/synthetics-analyze.asciidoc`](https://github.com/elastic/observability-docs/pull/2593/files#diff-42b4c8116873206244ac28435b5a0f1c55f7dbe7654d91176eb8a57863782eda) file, and the preview is here: [Analyze data from synthetic monitors](https://observability-docs_2593.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-analyze.html).

It is the Synthetics UI equivalent of the Uptime [Analyze](https://www.elastic.co/guide/en/observability/current/uptime-analyze.html) docs. I'm sure there is more details that we should add about interpreting data coming from synthetic monitors, but my current plan is to get some basic information up with this PR. Then we can add more details in separate PR(s). Let me know if you disagree with this plan!